FinSearchUnified/Subscriber/Frontend.php

```diff
@@ -106,12 +106,15 @@ public function onFrontendPreDispatch(Enlight_Event_EventArgs $args)
         if ($this->isSearchPage($request)) {
             Shopware()->Session()->offsetSet('isSearchPage', true);
             Shopware()->Session()->offsetSet('isCategoryPage', false);
+            Shopware()->Session()->offsetSet('isManufacturerPage', false);
         } elseif (StaticHelper::isCategoryPage($request)) {
             Shopware()->Session()->offsetSet('isCategoryPage', true);
             Shopware()->Session()->offsetSet('isSearchPage', false);
-        } elseif ($this->isManufacturerPage($request) || !$this->isWhiteListed($request)) {
+            Shopware()->Session()->offsetSet('isManufacturerPage', false);
+        } elseif (StaticHelper::isManufacturerPage($request) || !$this->isWhiteListed($request)) {
             Shopware()->Session()->offsetSet('isCategoryPage', false);
             Shopware()->Session()->offsetSet('isSearchPage', false);
+            Shopware()->Session()->offsetSet('isManufacturerPage', StaticHelper::isManufacturerPage($request));
```

Comment:
Just realised, that we will need to do this:
```suggestion
        } elseif (StaticHelper::isManufacturerPage($request)) {
            Shopware()->Session()->offsetSet('isCategoryPage', false);
            Shopware()->Session()->offsetSet('isSearchPage', false);
            Shopware()->Session()->offsetSet('isManufacturerPage', true);
        } elseif (!$this->isWhiteListed($request)) {
            Shopware()->Session()->offsetSet('isCategoryPage', false);
            Shopware()->Session()->offsetSet('isSearchPage', false);
            Shopware()->Session()->offsetSet('isManufacturerPage', false);
```

Reply:
We should catch this via test as well